### PR TITLE
Fix #231371 and #231364

### LINKED ISSUE #231371: Latest dev build timeout error same players. (old build works perfectly)
- [x] This issue isn't related to any plugins or plugin related crashes.
- [x] This issue isn't duplicated - you can check if it is by using the search bar located at the top left hand corner and select "Issues" on the left.
- [x] This issue includes appropriate markdown for sections - e.g. code blocks for crash dumps.
- [x] This issue is understandable - feel free to use your native language to write issues if you are not comfortable with English.
- [x] This issue matches the contributing guidelines - the link is just above the title field.

--------------------

1. join server.
2. wait building terrain
3. Timeout error in console, and player not login server.

kepkivagas

### PR DESCRIPTION
**Pull request by remotevase**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR matches the contributing guidelines - the link is just above the title field.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.
- [x] This PR has descriptive titles for commits - don't use "Updated <file-name>" as your title for all your commits in the PR.

--------------------
People could not join the server because when they attempted to, the client crashed. This allows people with under 512MB of RAM to join servers!

### CODE REVIEW

File: src/pocketmine/resources/pocketmine.yml [hunk 1]
```diff
@@ -103,8 +103,8 @@ level-settings:
  base-tick-rate: 1
  #Tick all players each tick even when other settings disallow this.
  always-tick-players: false
- #Set chunk radius for players, default as 30
- chunk-radius: 20
+ #Set chunk radius for players, default as 5
+ chunk-radius: 5
```
Comment (hoyinm14mc) [inline src/pocketmine/resources/pocketmine.yml]: If you have problem on this, you can edit your own pocketmine.yml file.

**Comment by hoyinm14mc** [comment]
You can just edit your own pocketmine.yml file if needed.

**Comment by remotevase** [comment]
Reason: tons of people are asking on forums, but don't know how/where to edit pocketmine.yml. this does it for them, making the process much easier

**Comment by Taki21** [comment]
No more irrelevant chnk radius issues since some people dont know how to do it :/

**Pull request merged**